frontend/packages/console-app/src/components/nodes/NodeLogs.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { Checkbox } from '@patternfly/react-core';
+import classnames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import {
+  Dropdown,
+  getQueryArgument,
+  LoadingBox,
+  setQueryArgument,
+} from '@console/internal/components/utils';
+import { modelFor, NodeKind, resourceURL } from '@console/internal/module/k8s';
+
+type NodeLogsProps = {
+  obj: NodeKind;
+};
+
+const NodeLogs: React.FC<NodeLogsProps> = ({ obj: node }) => {
+  const dropdownItems = {
```

Comment:
The issue is that this isn't a well-defined list. It's the path to a log file, which isn't restricted to these values. We might need to make this an editable select or a plain text input.